packages/react/src/components/FormattingToolbar/DefaultButtons/NestBlockButtons.tsx

```diff
@@ -9,9 +9,12 @@ import { formatKeyboardShortcut } from "@blocknote/core";
 export const NestBlockButton = <BSchema extends BlockSchema>(props: {
   editor: BlockNoteEditor<BSchema>;
 }) => {
-  const [canNestBlock, setCanNestBlock] = useState<boolean>();
+  const [canNestBlock, setCanNestBlock] = useState<boolean>(
+    props.editor.canNestBlock()
```

Comment:
I think we should use an initializer here (`() => ...`) otherwise this gets called every render